docs/migration_guide.md

```diff
@@ -33,134 +36,53 @@ When upgrading your existing Habor instance to a newer version, you may need to
-
-    docker run -it --rm -e DB_USR=root -v /data/notary-db/:/var/lib/mysql -v /data/database:/var/lib/postgresql/data goharbor/harbor-migrator:${tag} --db up
-
-    docker run -it --rm -v /data/clair-db/:/clair-db -v /data/database:/var/lib/postgresql/data goharbor/harbor-migrator:${tag} --db up
-    ```
+6. Upgrade `harbor.cfg`.
 
     **NOTE:** The ${harbor_cfg} will be overwritten, you must move it to your installation directory after migration.
 
     ```
     docker run -it --rm -v ${harbor_cfg}:/harbor-migration/harbor-cfg/harbor.cfg goharbor/harbor-migrator:[tag] --cfg up
     ```
+    
+    **NOTE:** The schema upgrade and data migration of Database is performed by adminserver when Harbor starts, if the migration fails, 
+    please check the log of adminserver to debug.
 
-     **NOTE:** Some errors like
-
-    ```
-    [ERROR] Missing system table mysql.roles_mapping; please run mysql_upgrade to create it
-    [ERROR] Incorrect definition of table mysql.event: expected column 'sql_mode' at position ... ...
-    [ERROR] mysqld: Event Scheduler: An error occurred when initializing system tables. Disabling the Event Scheduler.
-    [Warning] Failed to load slave replication state from table mysql.gtid_slave_pos: 1146: Table 'mysql.gtid_slave_pos' doesn't exist
-    ```
-    will be occurred during upgrading from harbor 1.2 to harbor 1.3, just ignore them if harbor can start successfully.
-
-    ```
-    /usr/lib/python2.7/site-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
-    ```
-    will be occurred during upgrading from harbor <= v1.5.0 to harbor v1.6.0, just ignore them if harbor can start successfully.
 
-7. Under the directory `./harbor`, run the `./install.sh` script to install the new Harbor instance. If you choose to install Harbor with components like Notary and/or Clair, refer to [Installation & Configuration Guide](../docs/installation_guide.md) for more information.
+7. Under the directory `./harbor`, run the `./install.sh` script to install the new Harbor instance. If you choose to install Harbor with components like Notary, Clair, and chartmuseum, refer to [Installation & Configuration Guide](../docs/installation_guide.md) for more information.
 
 ### Roll back from an upgrade
 For any reason, if you want to roll back to the previous version of Harbor, follow the below steps:
 
-**NOTE:** Roll back doesn't support upgrade across v1.5.0, like from v1.2.0 to v1.6.0. It's because Harbor changes DB to Postgresql from v1.6.0, the migrator cannot roll back data to MariaDB.    
+**NOTE:** Roll back doesn't support upgrade across v1.5.0, like from v1.2.0 to v1.7.0. It's because Harbor changes DB to Postgresql from v1.7.0, the migrator cannot roll back data to MariaDB.    
```

Comment:
Wording:

* s/It's because/This is because/
* s/Postgresql/PostgreSQL/

Reply:
Done